http-request-response.sublime-syntax

```diff
@@ -8,8 +8,8 @@ file_extensions:
 variables:
   content_type_sep: (?=;|$)
   multipart_form_data_boundary: (?:^---+.*$)
-  http_version: (?:\bHTTP/\d\.\d\b)
-  end_of_body: '^{{http_version}}\s\d{3}\s\w+'
+  http_version: (?:\bHTTP/\d(?:\.\d)?\b)
+  end_of_body: '^{{http_version}} \d{3}(?: \w+)?'
```

Comment:
This isn't quite perfect as we seem to really want a reason phrase after the status code (which curl's synthetized output does not have), and on the other hand the same curl's synthetized output has just a one trailing space without the message after the status code (e.g. `HTTP/2 200 `).

I believe because of this, the first header following such a status line gets colorized incorrectly:
![Screenshot from 2021-07-31 11-45-19](https://user-images.githubusercontent.com/109152/127734419-a400c8c5-6cef-45d3-8276-44e49848e809.png)

I tried a few end_of_body alternatives but couldn't figure out what would take in a few minutes, maybe it needs to be addressed somewhere else. Anyway I think this would be fine already as is, not that much a biggie.

Reply:
Thanks for pointing this out, I will merge it and see if I can improve upon it :)